src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeProtocol.cs

```diff
@@ -142,6 +164,12 @@ public static bool TryParseResponseMessage(ref ReadOnlySequence<byte> buffer, ou
             }
         }
 
+        /// <summary>
+        /// Creates a new <see cref="HandshakeRequestMessage"/> using the specified bytes.
```

Comment:
Creates a new [HandshakeResponseMessage] **from the specified serialized representation** ~~using the specified bytes~~.